src/main/java/com/google/codeu/servlets/AboutServlet.java

```diff
@@ -0,0 +1,120 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+
+
```

Comment:
Looks like this file has a lot of extra newlines. Some newlines are good to help separate code blocks, but right now I think these extra newlines make it harder to read. Do you mind getting rid of the extra newlines and making sure your code is properly indented to make it easier to read? Thanks!